.github/workflows/test.yml

```diff
@@ -42,7 +42,7 @@ jobs:
             name: test with OCB
     services:
       postgres:
-        image: postgres:9.6
+        image: ursa/postgresql:latest
```

Comment:
Is this an appropriate change?

Also, shouldn't we be using postgis, which has an official image over on docker hub
There is no maintained 9.6 image, but the oldest still maintaned image is [postgis/postgis:10-3.2](https://registry.hub.docker.com/r/postgis/postgis/tags?page=1&name=10-3.2)

Reply:
@aleuffre Yes it is appropriate, even if your solution is better.